template/overall_header.html

```diff
@@ -26,7 +26,7 @@
 <!--
 	phpBB style name: prosilver
 	Based on style:   prosilver (this is the default phpBB3 style)
-	Original author:  Tom Beddard ( http://www.subBlue.com/ )
+	Original author:  Tom Beddard ( }://www.subBlue.com/ )
```

Comment:
Isn't this a typo?

Reply:
Yeah, didn't notice I did that. Will fix. Thanks for noticing! 👍